src/renderers/shaders/ShaderLib/meshphysical_frag.glsl.js

```diff
@@ -3,7 +3,6 @@ export default /* glsl */`
 
 #ifdef PHYSICAL
 	#define IOR
-	#define CLEARCOAT
 	#define SPECULAR
 #endif
```

Comment:
@WestLangley I think we should get rid of all this `PHYSICAL` ifdef logic. Do you agree?